src/main/java/com/wcc/platform/service/CmsService.java

```diff
@@ -168,4 +169,18 @@ public AboutUsPage getAboutUs() {
 
     throw new ContentNotFoundException(PageType.ABOUT_US);
   }
+
+  /**
+   * Read JSON and convert to Pojo CelebrateHerPage.
+   *
+   * @return Pojo CelebrateHerPage.
+   */
+  public CelebrateHerPage getCelebrateHer() {
+    try {
+      final var data = FileUtil.readFileAsString(PageType.CELEBRATE_HER.getFileName());
```

Comment:
We already have database and it would be nice to use it for new pages from start.